src/test/java/uk/gov/register/functional/EntriesResourceFunctionalTest.java

```diff
@@ -25,30 +26,32 @@
 
 public class EntriesResourceFunctionalTest {
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
+    private WebTarget addressTarget;
 
     @Before
     public void setup() {
         register.wipe();
+        addressTarget = register.targetRegister("address");
```

Comment:
Is this required to be run before each test? Maybe just maybe we can make it nicely into:
`private final WebTarget addressTarget = register.targetRegister("address");`
what do you think? It does have `final` on it too ;)